### ISSUE TITLE
[Youtube_Get_Video_Info]java.io.FileNotFoundException: https://www.youtube.com/get_video_info? even there is html5=1 as param

### ISSUE THREAD

**Issue by suryajanaki** [HaarigerHarald/android-youtubeExtractor#183]
There is a FileNotFoundException even if we add the html5=1 to the URL. 
From past few days this exception is throwing when we extract the video.

**Comment by kishanverma26** [comment]
> There is a FileNotFoundException even if we add the html5=1 to the URL.
> From past few days this exception is throwing when we extract the video.

same issue with me

**Comment by pongsatornw** [comment]
Same with me too, even if we add html=1 to URL, it can no longer play a video.

ps. sorry for my bad english.

**Comment by grizzly3ear** [comment]
Not working for me too 😥

According to the latest fix that is resolve by add html5=1 to query string, how can we know that we should add html5=1 ?

**Comment by kalasour** [comment]
It loading for a long time pls help 😭

**Comment by williamycyh** [comment]
I think it is User-Agent's problem.
Modify the below User-Agent to Mac, sometimes it works, sometimes it doesn’t work. Hope someone can solve it completely.
Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/40.0.2214.115 Safari/537.36

**Comment by suryajanaki** [comment]
@williamycyh 
That is correct but it is not working for every time. It seems they might have changes the file location.

**Comment by grizzly3ear** [comment]
First of all, I found that youtube api trying to set cookies with key ```YSC``` / ```VISITOR_INFO1_LIVE```

On my project, I've already tried to remove the cookies before send the request to get_video_info api which is always work for me :)

You can try on your incognito browser mode with empty cookies on domain ```.youtube.com```

Hope that helps :)

**Comment by SushilBerwal** [comment]
> There is a FileNotFoundException even if we add the html5=1 to the URL.
> From past few days this exception is throwing when we extract the video.

Same issue with me. any solution

**Comment by saurabhgoyal795** [comment]
any changes??

**Comment by saurabhgoyal795** [comment]
how do we solve this problem??

**Comment by SushilBerwal** [comment]
YouTube live video not playing but recorded play

**Comment by karthikstrk** [comment]
@williamycyh did you fix that? Can you share ur code. Please!

**Comment by LoyalPotato** [comment]
https://stackoverflow.com/questions/67615278/get-video-info-youtube-endpoint-suddenly-returning-404-not-found

Adding `&c=TVHTML5&cver=6.20180913` like said above fixed the issue of not getting anything from the get_video_info endpoint

**Comment by shailendra-rajpoot** [comment]
> https://stackoverflow.com/questions/67615278/get-video-info-youtube-endpoint-suddenly-returning-404-not-found
> 
> Adding `&c=TVHTML5&cver=6.20180913` like said in the link fixed the issue of not getting anything from the get_video_info endpoint

after doing this is not working today plz resolve it
